xml/deployment_image.xml

```diff
@@ -56,7 +56,7 @@
    up and clones can be used immediately.
   </para>
   <para>
-   The <command>clone-master-clean-up</command> command for example removes:
```

Comment:
To prevent interrupting the train of thought, I would move 'For example,' to the beginning of the sentence.